pep_sphinx_extensions/pep_theme/static/mq.css

```diff
@@ -2,25 +2,10 @@
 
 /* Media Queries */
 
-/* Further reduce width of fixed elements for smallest screens */
-@media (max-width: 32em) {
-    section#pep-page-section {
-        padding: 0.25rem;
-    }
-    dl.footnote > dt,
-    dl.footnote > dd {
-        padding-left: 0;
-        padding-right: 0;
-    }
-    pre {
-        font-size: 0.75rem;
-    }
-}
```

Comment:
Are these rules fine to remove? I'm not able to use the browser responsive tester at the moment.

Reply:
Yes, because the `@media (max-width: 40em)` takes precedence and the `@media (max-width: 32em)` is never applied.

Demo: https://jsfiddle.net/9okLgwz6/

Reply:
At least all the rules except the following are in the `@media (max-width: 40em)` block:

```css
    dl.footnote > dt,
    dl.footnote > dd {
        padding-left: 0;
        padding-right: 0;
    }
```

So this is a candidate to keep. But I can't find any matching HTML to check?

Reply:
I've looked locally and concur, thanks.